components/ingest-service/pipeline/processor/runs_project_tagger.go

```diff
@@ -125,7 +125,7 @@ func nodeMatchesAllConditions(node backend.Node, conditions []*iam_v2.Condition)
 			if !foundMatch {
 				return false
```

Comment:
(nothing to do with this PR just a question to myself and @lancewf) 

Do we need this `foundMatch` logic? why not just return `true` in like `121` like the other functions? (I might be missing something)